### PR TITLE
performance-addon-operators: add new binary

### PR DESCRIPTION
**Pull request by marcel-apf**
Add the performance-profile-creator binary to the PAO image.

Signed-off-by: Marcel Apfelbaum <marcel@redhat.com>

### CODE REVIEW

**Review by fromanirh — COMMENTED** [review]
/lgtm

File: ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml [hunk 1]
```diff
@@ -27,6 +27,8 @@ images:
       paths:
       - destination_dir: .
         source_path: /go/src/github.com/openshift-kni/performance-addon-operators/build/_output/bin/performance-addon-operators
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift-kni/performance-addon-operators/build/_output/bin/performance-profile-creator
```
Comment (fromanirh) [inline ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml:31]: I was too hasty with my lgtm. It's very possivle we will need a new `input` definition. Need to check.

**Review by fromanirh — COMMENTED** [review]
/lgtm cancel

**Review by swatisehgal — APPROVED** [review]
/lgtm

**Pull request merged**